src/items/LinkItem.js

```diff
@@ -20,6 +38,8 @@ const LinkItem = ({
 }) => {
   const classes = useStyles();
 
+  const [loaded, setLoaded] = useState(false);
```

Comment:
Usually boolean starts with `is` or `has` maybe `isLoading` would be a better boolean name (and default to `true`)